code/modules/mob/living/carbon/human/examine.dm

```diff
@@ -52,7 +52,16 @@
 				t_his = "her"
 				t_him = "her"
 
-	msg += "<EM>[src.name]</EM>!\n"
+	msg += "<EM>[src.name]</EM>"
+
+	var/list/nospecies = list("Abductor", "Shadowling", "Neara", "Monkey", "Stok", "Farwa", "Wolpin") //species that won't show their race no matter what
+
+	if (skipjumpsuit && skipface || (src.species.name in nospecies)) //either obscured or on the nospecies list
+		msg += "!\n"	//omit the species when examining
+	else if (src.species.name == "Slime People") //snowflakey because Slime People are defined as a plural
+		msg += ", a Slime Person!\n"

```

Comment:
Literally every single race has their plural and singular nouns done right, but Slime People have "Slime People" both under singular and plural.

I'm not sure if changing the name to a proper, singular one will be okay with the rest of the code, especially the the SQL.